build.gradle

```diff
@@ -869,6 +869,9 @@ project(':iceberg-orc') {
 }
 
 project(':iceberg-parquet') {
+  apply plugin: 'java-library'
```

Comment:
is this one really needed? this should have been applied to all subprojects already